README

```diff
@@ -2,8 +2,7 @@
 
 [![Build Status](https://travis-ci.org/libunwind/libunwind.svg?branch=master)](https://travis-ci.org/libunwind/libunwind)
 
-This is version 1.4 of the unwind library.  This library supports
-several architecture/operating-system combinations:
+This library supports several architecture/operating-system combinations:
```

Comment:
@Arfrever, @zhaofengli, how do you feel about a brief introduction line (copied from https://www.nongnu.org/libunwind/):
```suggestion
The primary goal of this project is to define a portable and efficient C programming interface (API) to determine the call-chain of a program.

This library supports several architecture/operating-system combinations:
```